imap_processing/cdf/config/imap_mag_l1_variable_attrs.yaml

```diff
@@ -7,25 +23,34 @@ default_attrs: &default
   VALIDMIN: -9223372036854775808
   VALIDMAX: 9223372036854775807
   VAR_TYPE: data
+  UNITS: ' '
+
+default_coords: &default_coords
+  # Assumed values for all coordinate attrs unless overwritten
+  FORMAT: F2.4  # Float with 4 digits
+  VAR_TYPE: data
+  DISPLAY_TYPE: time_series
+  FILLVAL: -9223372036854775808
+  VALIDMIN: -9223372036854775808
+  VALIDMAX: 9223372036854775807
+  UNITS: counts
 
-mag_raw_vector_attrs:
-  <<: *default
+raw_vector_attrs:
+  <<: *default_coords
   CATDESC: Raw unprocessed magnetic field vector data in bytes
   DEPEND_1: direction
   FIELDNAM: Magnetic Field Vector
   LABLAXIS: Raw binary magnetic field vector data
   FORMAT: I3
-  UNITS: counts
 
 vector_attrs:
-    <<: *default
+    <<: *default_coords
```

Comment:
Is this `_vector_attrs` coordinate data array or science data array? I tried to look for `xr.Dataset(..., coords={..}` that uses this but couldn't find it. I may have been confused because you are using `default_coords`. If it's science data array, can you use different default value?

Reply:
It's a coordinate data array. Maybe we can come up with some standard way of designating attributes as for coordinates because I'm hearing it as a form of confusion. I standardized on using "attrs" for coordinates but that's a terrible standard, really